packages/gatsby-theme-docs/gatsby-config.js

```diff
@@ -210,6 +213,54 @@ module.exports = (themeOptions = {}) => {
       'gatsby-plugin-remove-trailing-slashes',
       'gatsby-plugin-meta-redirect',
       'gatsby-plugin-netlify-cache',
+      {
```

Comment:
The feed generator needs to go above the plugins that explicitly have a comment "the following plugins need to be last" above them.